addons/viewdistance/XEH_postInit.sqf

```diff
@@ -5,17 +5,7 @@ if (!hasInterface) exitWith {};
 ["SettingsInitialized", {
     // if not enabled, then bugger off.
     if !(GVAR(enabled)) exitWith {};
-    
-    // Force the view distance down to the limit.
-    if (viewDistance > GVAR(limitViewDistance)) then {
-        setViewDistance GVAR(limitViewDistance);
-    };
-    
-    // Adapt view distance when the player is created or changed according to whether client is on foot or vehicle.
-    ["playerChanged",{
-        [false] call FUNC(adaptViewDistance);
-    }] call EFUNC(common,addEventHandler);
-    
```

Comment:
If you do end up preserving it, it'd probably make more sense for it to be `[false] call FUNC(adaptViewDistance)` anyway :+1: Otherwise all looks good